src/autolabel/tasks/attribute_extraction.py

```diff
@@ -120,8 +120,19 @@ def construct_prompt(
 
         input[self.OUTPUT_DICT_KEY] = ""
 
+        # check if all mapped keys in input are in the example template
+        try:
+            current_example = example_template.format(**input)
+        except KeyError as e:
+            curr_text_prompt = example_template.format_map(defaultdict(str, input))
```

Comment:
we should be updating current_example as well right? I think it being used in the trim_prompt function below